src/components/container/LabResultsBloodType/LabResultsBloodType.tsx

```diff
@@ -0,0 +1,165 @@
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import React, { useEffect, useState } from "react";
+import { Action, LoadingIndicator } from "../../presentational";
+import "./LabResultsBloodType.css";
+import "../HealthPreviewSection/HealthPreviewSection.css"
+import language from "../../../helpers/language";
+import { bloodTypeLabs, singleLabs, manyLabs } from "./LabResultsBloodType.previewdata";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+import iconBloodType from "./icon-bloodtype.svg";
+
+export interface LabResultsBloodTypeProps {
+    previewState?: "BloodTypeLabs" | "SingleLabs" | "ManyLabs" | "NoData"
+    summaryResultsOnly?: true | false
+    maximumResults?: number
+    onClick?: any
+    externalInfoLink?: true | false
+    externalInfoLinkDefault?: string
+    externalInfoLinkSpanish?: string
+}
+
+export default function (props: LabResultsBloodTypeProps) {
+    const [model, setModel] = useState<any>(null);
+    const [noOverflow, setNoOverflow] = useState<boolean>(false);
```

Comment:
This can be removed if not used

Reply:
Updated to remove all references to noOverflow